### DISCUSSION TITLE
Error: The values of --overrides does not seem to be valid JSON. [Windows]

### DISCUSSION THREAD

**Discussion by sealabr** [artilleryio/artillery#1231] [Artillery]
Hello I'm trying to implement artillery inside nodejs with shelljs.

my idea is to use pm2 as a cron service to perform daily tests for my application.

but this command does not run within windows, only on linux, is there any solution?

```
const shell = require('shelljs');
var path = require('path');
var cmd = `artillery run --overrides '{"config":{"phases":[{"duration":60,"arrivalRate":0,"rampTo":10,"name":"teste"}]}}' -o ${__dirname}/droz/reports/example.json ${__dirname}/droz/example.yml`;
shell.exec(cmd, { async: true });
```

Error:
Error: The values of --overrides does not seem to be valid JSON.

![image](https://user-images.githubusercontent.com/21187479/142046158-a17be090-a1fc-44b5-8ad3-7263e88d3be9.png)
pm2 imagem

**Comment by hassy** [comment]
hi @sealabr 👋 please see the Windows-specific notes here: https://www.artillery.io/docs/guides/guides/command-line#override-parts-of-the-test-script-on-the-fly
